### ISSUE TITLE
Can't render the simplest project I can imagine...

### ISSUE THREAD

**Issue by pmorch** [codesandbox/codesandbox-client#2078]
## Description of the problem

I have index.html (9 lines) and no js files and no dependencies. Nothing gets rendered.

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>codesandbox</title>
  </head>
  <body>
    Page
  </body>
</html>
```

![image](https://user-images.githubusercontent.com/159292/59862490-29a21e00-9383-11e9-9bcf-925feeb6be58.png)

## How has this issue affected you? What are you trying to accomplish?

I'm trying to do lots of things, but when even index.html doesn't work I'm lost.

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | I have no idea.. The version running at codesandbox.io
| Browser          | FF 67.0.2 and chrome 75.0.3770.100
| Operating System | Linux Xubuntu 18.04

**Comment by lbogdan** [comment]
Hey @pmorch ,

This is most probably an issue with rendering static HTML in `vue-cli` sandboxes. While this is a bug, what exactly is your end goal here?

**Comment by lbogdan** [comment]
Seems like a duplicate of #2010 .

**Comment by pmorch** [comment]
Ok, so here is the road that got me here:

I was trying to get [ref](https://vuejs.org/v2/api/#ref) working in Vue in Codesandbox. I've now learned that for some reason ref works if a component is defined inline in main.js, but it doesn't work if the component is defined in a Whatever.vue file. See [this sandbox](https://codesandbox.io/s/inline-vs-vue-file-63h6v). If you hit the rendered link from a component in App.vue, you'll see an error in the console:

> [Vue warn]: Property or method "children" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. See: https://vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.

If you uncomment this line:

    // App = AppInline;

Hitting that almost-same link from the almost-identical inline implementation works and console.log logs to the console just fine.

I've downloaded the sandbox (neat .zip download feature...) and built it locally with my local webpack setup, and both versions work fine here. So there is something going on where some Vue things don't work when in .vue files in Codesandbox. That is a separate issue that I haven't logged anywhere.

During the course of narrowing down that issue, I got to a place where all I saw was a full-screen white screen in the preview. I then tried to narrow *that* down (assuming it was a different symptom of the same issue, I guess). Leading me to an ever simpler sandbox until there was nothing left and it still didn't work. I've forgotten exactly at what point I transitioned from seeing one issue to seeing the next.

Additional feedback: Usability would *greatly* improve if it was obvious when the preview has caught up with the saved code. It can take many seconds to get there and so I'm never really 100% sure whether I'm seeing a stable up-to-date version in the preview, or whether it's going to reload in 2 seconds. I found myself changing constant strings in the code or markup just to make sure the GUI changed accordingly so I knew I was seeing fresh rendering. When changing from one sandbox to another, I have also seen that the preview blinks with renderings from another sandbox that I used to be editing a long time ago. Also not something that gives me confidence. Also, sometimes I need to save the files and/or hit refresh in the preview window. Sometimes not.

That, coupled with a real error where the screen stayed full-screen white made for a frustrating experience....
